doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>related to these titles and sections I menioned above, I would have the concat and the extend examples in different sections, or maybe in the same section but different subsection. I would not compare them here, only show concat to extend with new chains, extend to add posterior_predictive to an already existing inferencedata with posterior samples (so different example), nothing else. Users at this level don't need to know more, more advanced guides or the docstring could be a place to discuss the differences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>